wallet/src/de/schildbach/wallet/ui/main/WalletTransactionsFragment.kt

```diff
@@ -49,7 +49,7 @@ class WalletTransactionsFragment : Fragment(R.layout.wallet_transactions_fragmen
         private const val HEADER_ITEM_TAG = "header"
     }
 
-    private val viewModel by viewModels<MainViewModel>()
+    private val viewModel by activityViewModels<MainViewModel>()
```

Comment:
This was creating a separate MainViewModel instance, so we kept two in memory. Not good.